scripts/update_lit_checks.py

```diff
@@ -31,9 +31,15 @@
 script_name = os.path.basename(__file__)
 NOTICE = (f';; NOTE: Assertions have been generated by {script_name} and ' +
           'should not be edited.')
+
+
 RUN_LINE_RE = re.compile(r'^\s*;;\s*RUN:\s*(.*)$')
 CHECK_PREFIX_RE = re.compile(r'.*--check-prefix[= ](\S+).*')
-FUNC_RE = re.compile(r'(^\s*)\(func \$(\S*).*$', re.MULTILINE)
+
+items = ['type', 'import', 'global', 'memory', 'data', 'table', 'elem', 'tag',
+         'export', 'start', 'func']
```

Comment:
can we find all of these based on their indentation, as opposed to a hardcoded list?

Reply:
In principle, yes, but I think it would be complex enough that it would be better to go with the list. This script is already more complex than I would like it to be :/